app/models/document.rb

```diff
@@ -109,7 +109,7 @@ def self.from_publishing_api(payload)
       content_id: payload['content_id'],
       title: payload['title'],
       summary: payload['description'],
-      body: payload['details']['body'],
+      body: payload['details']['body'][0]['content'],
```

Comment:
Can you be sure the first element is always going to be govspeak.I remember is share-sale-publisher, we did something like this:
```
{ |format| format[:content_type] == "text/govspeak" }[:content]
```